crates/rethnet_evm/src/state/fork.rs

```diff
@@ -123,19 +168,17 @@ impl revm::Database for ForkDatabase {
     }
 }
 
-impl revm::DatabaseCommit for ForkDatabase {
+impl revm::DatabaseCommit for ForkState {
     fn commit(&mut self, changes: HashMap<Address, Account>) {
-        self.layered_db.commit(changes)
+        self.layered_state.commit(changes)
     }
 }
 
-impl crate::DatabaseDebug for ForkDatabase {
-    type Error = ForkDatabaseError;
+impl crate::state::debug::StateDebug for ForkState {
```

Comment:
`use crate::state::debug::StateDebug` at the top of the file. This also removes the necessity for the verbose usage of full paths to call trait functions